account_invoice_comment_template/models/account_move.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2014 Guewen Baconnier (Camptocamp SA)
+# Copyright 2013-2014 Nicolas Bessi (Camptocamp SA)
+# Copyright 2018 Tecnativa - Vicent Cubells
+# Copyright 2019 Iván Todorovich (Druidoo)
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _name = "account.move"
+    _inherit = ["account.move", "comment.template"]
+
+    # Deprecated, these fields will be removed in the future (15.0 probably)
+    note1 = fields.Html(string="Top Comment", readonly=True)
```

Comment:
Why not removing it? We can discuss in v14 for its removal, but I would remove it here for now.